cmd/monaco/integrationtest/v2/all_configs_integration_test.go

```diff
@@ -19,52 +19,49 @@
 package v2
 
 import (
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/internal/test"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/featureflags"
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/testutils"
-	"testing"
-
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
 	"github.com/spf13/afero"
-	"gotest.tools/assert"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
 )
 
-// tests all configs for a single environment
-func TestIntegrationAllConfigsClassic(t *testing.T) {
-	specificEnvironment := "classic_env"
-
-	runAllConfigsTest(t, specificEnvironment)
-}
-
-func TestIntegrationAllConfigsPlatform(t *testing.T) {
-	specificEnvironment := "platform_env"
-
-	runAllConfigsTest(t, specificEnvironment)
-}
-
-func runAllConfigsTest(t *testing.T, specificEnvironment string) {
-	configFolder := "test-resources/integration-all-configs/"
-	manifest := configFolder + "manifest.yaml"
-
-	envVars := map[string]string{featureflags.Experimental().EnvName(): "true"}
-
-	RunIntegrationWithCleanupGivenEnvs(t, configFolder, manifest, specificEnvironment, "AllConfigs", envVars, func(fs afero.Fs, _ TestContext) {
-
-		// This causes a POST for all configs:
-
-		cmd := runner.BuildCli(fs)
-		cmd.SetArgs([]string{"deploy", "--verbose", manifest, "--environment", specificEnvironment})
-		err := cmd.Execute()
-
-		assert.NilError(t, err)
-
-		// This causes a PUT for all configs:
-
-		cmd = runner.BuildCli(fs)
-		cmd.SetArgs([]string{"deploy", "--verbose", manifest, "--environment", specificEnvironment})
-		err = cmd.Execute()
-		assert.NilError(t, err)
-
-	})
+func TestAllConfigs(t *testing.T) {
+	tests := []struct {
+		name, environment string
+	}{
+		{
+			name:        "tests all known configs against classic url",
+			environment: "classic_env",
+		},
+		{
+			name:        "tests all known configs against platform url",
+			environment: "platform_env",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			configFolder := "test-resources/integration-all-configs/"
+			manifest := configFolder + "manifest.yaml"
+
+			envVars := map[string]string{featureflags.Experimental().EnvName(): "true"}
+
+			RunIntegrationWithCleanupGivenEnvs(t, configFolder, manifest, tc.environment, "AllConfigs", envVars, func(fs afero.Fs, _ TestContext) {
+				{
+					// This causes a POST for all configs:
+					_, err := test.Monacof("deploy %s --environment %s", manifest, tc.environment).WithFs(fs).Run()
```

Comment:
nit/thought: this change isn't really related to this commit, I think they should have gone in the refactor commit - having these additional changes makes it a bit harder to review the actual change.